tests/Console/ModelsCommand/LaravelCustomCasts/__snapshots__/Test__test__1.php

```diff
@@ -28,8 +28,9 @@
  * @property \Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\LaravelCustomCasts\Casts\CastedProperty $casted_property_with_return_docblock
  * @property \Barryvdh\LaravelIdeHelper\Tests\Console\ModelsCommand\LaravelCustomCasts\Casts\CastedProperty $casted_property_with_return_docblock_fqn
  * @property array $casted_property_with_return_primitive
- * @property array|null $casted_property_with_return_primitive_docblock
- * @property array|null $casted_property_with_return_nullable_primitive
+ * @property array $casted_property_with_return_primitive_docblock
+ * @property array $casted_property_with_return_nullable_primitive
```

Comment:
Why is this changed? Is it intentional based on the fix? Thanks

Reply:
Yes, this is intentional. I didn't change the migration for these properties and they weren't marked as `nullable()`, so the `|null` gets removed by the fix in this PR.